pkg/stomp/StompContext.php

```diff
@@ -173,6 +180,8 @@ public function createConsumer(Destination $destination): Consumer
     {
         InvalidDestinationException::assertDestinationInstanceOf($destination, StompDestination::class);
 
+        $this->transient = false;
```

Comment:
As soon as the first consumer is created, we go long-lived and disable all transient functionality so that queues with heartbeats don't have their connections pulled out from under them.